cmake/configs/default.cmake

```diff
@@ -457,6 +457,7 @@ hunter_default_version(xrender VERSION 0.9.7)
 hunter_default_version(xshmfence VERSION 1.3)
 hunter_default_version(xtrans VERSION 1.2.7)
 hunter_default_version(xxf86vm VERSION 1.1.2)
+hunter_default_version(xxhash VERSION v0.6.5-p1)
```

Comment:
Usually, in this file it's without `v`, can you change `v0.6.5-p1` to `0.6.5-p1` please?